protocol/x/clob/keeper/grpc_stream_finalize_block.go

```diff
@@ -0,0 +1,49 @@
+package keeper
+
+import (
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/telemetry"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/dydxprotocol/v4-chain/protocol/lib/metrics"
+	"github.com/dydxprotocol/v4-chain/protocol/x/clob/types"
+)
+
+// Returns the order updates needed to update the fill amount for the orders
+// from local ops queue, according to the latest onchain state (after FinalizeBlock).
+func (k Keeper) getUpdatesToSyncLocalOpsQueue(
+	ctx sdk.Context,
+) *types.OffchainUpdates {
+	localValidatorOperationsQueue, _ := k.MemClob.GetOperationsToReplay(ctx)
+	fetchOrdersInvolvedInOpQueue(localValidatorOperationsQueue)
+	orderIdsFromLocal := fetchOrdersInvolvedInOpQueue(
+		localValidatorOperationsQueue,
```

Comment:
Think we need to monitor this to make sure the removal of 
```
		orderIdsFromProposed := fetchOrdersInvolvedInOpQueue(
			operations,
		)
```
will not break logic